internal/ir/xds.go

```diff
@@ -355,6 +357,14 @@ type OIDC struct {
 	Scopes []string `json:"scopes,omitempty" yaml:"scopes,omitempty"`
 }
 
+// BasicAuth defines the schema for the HTTP Basic Authentication.
+//
+// +k8s:deepcopy-gen=true
+type BasicAuth struct {
+	// The username-password pairs in htpasswd format.
+	Users string `json:"users,omitempty" yaml:"users,omitempty"`
```

Comment:
should this be []byte ?

Reply:
also needs to be unreadable, need to add logic in `Printable`

Reply:
It can be changed to []byte.